Incorrect error page in dev mode when quarkus-kubernetes is in the classpath

Create a project with `quarkus-kubernetes` as extension.

Then start in dev mode.

Then create a new java file and forget to add a mandatory import (e.g. `List getAll()` without importing `java.util;.List`). You then get the following nasty error


The web page shows

```
java.lang.RuntimeException: Compilation failed[warning: No SupportedSourceVersion annotation found on io.sundr.transform.internal.VelocityTransformationProcessor, returning RELEASE_6., warning: Supported source version 'RELEASE_6' from annotation processor 'io.sundr.transform.internal.VelocityTransformationProcessor' less than -source '1.8', warning: No SupportedSourceVersion annotation found on io.sundr.resourcecify.internal.processor.ResourcecifyProcessor, returning RELEASE_6., warning: Supported source version 'RELEASE_6' from annotation processor 'io.sundr.resourcecify.internal.processor.ResourcecifyProcessor' less than -source '1.8', warning: No SupportedSourceVersion annotation found on io.sundr.builder.internal.processor.BuildableProcessor, returning RELEASE_6., warning: Supported source version 'RELEASE_6' from annotation processor 'io.sundr.builder.internal.processor.BuildableProcessor' less than -source '1.8', warning: No SupportedSourceVersion annotation found on io.sundr.builder.internal.processor.ExternalBuildableProcessor, returning RELEASE_6., warning: Supported source version 'RELEASE_6' from annotation processor 'io.sundr.builder.internal.processor.ExternalBuildableProcessor' less than -source '1.8', /work/fruits-app/src/main/java/com/example/FruitResource.java:36: error: cannot find symbol

The stacktrace below has been reversed to show the root cause first. Click Here to see the original stacktrace

java.lang.RuntimeException: Compilation failed[warning: No SupportedSourceVersion annotation found on io.sundr.transform.internal.VelocityTransformationProcessor, returning RELEASE_6., warning: Supported source version 'RELEASE_6' from annotation processor 'io.sundr.transform.internal.VelocityTransformationProcessor' less than -source '1.8', warning: No SupportedSourceVersion annotation found on io.sundr.resourcecify.internal.processor.ResourcecifyProcessor, returning RELEASE_6., warning: Supported source version 'RELEASE_6' from annotation processor 'io.sundr.resourcecify.internal.processor.ResourcecifyProcessor' less than -source '1.8', warning: No SupportedSourceVersion annotation found on io.sundr.builder.internal.processor.BuildableProcessor, returning RELEASE_6., warning: Supported source version 'RELEASE_6' from annotation processor 'io.sundr.builder.internal.processor.BuildableProcessor' less than -source '1.8', warning: No SupportedSourceVersion annotation found on io.sundr.builder.internal.processor.ExternalBuildableProcessor, returning RELEASE_6., warning: Supported source version 'RELEASE_6' from annotation processor 'io.sundr.builder.internal.processor.ExternalBuildableProcessor' less than -source '1.8', /work/fruits-app/src/main/java/com/example/FruitResource.java:36: error: cannot find symbol
    public List<FruityVice> getAll() {
           ^
  symbol:   class List
  location: class com.example.FruitResource]
	at io.quarkus.dev.JavaCompilationProvider.compile(JavaCompilationProvider.java:59)
	at io.quarkus.dev.ClassLoaderCompiler.compile(ClassLoaderCompiler.java:181)
	at io.quarkus.dev.RuntimeUpdatesProcessor.checkForChangedClasses(RuntimeUpdatesProcessor.java:186)
	at io.quarkus.dev.RuntimeUpdatesProcessor.doScan(RuntimeUpdatesProcessor.java:121)
	at io.quarkus.vertx.http.deployment.devmode.VertxHotReplacementSetup$1.handle(VertxHotReplacementSetup.java:51)
	at io.quarkus.vertx.http.deployment.devmode.VertxHotReplacementSetup$1.handle(VertxHotReplacementSetup.java:43)
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$2(ContextImpl.java:316)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)

```

Does annotation processors leaks somehow. CC @geoand

--- a/extensions/kubernetes/deployment/pom.xml
+++ b/extensions/kubernetes/deployment/pom.xml
@@ -23,6 +23,10 @@
             <classifier>noapt</classifier>
             <exclusions>
                 <exclusion>
+                    <groupId>io.sundr</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+                <exclusion>
                     <groupId>com.sun</groupId>
                     <artifactId>tools</artifactId>
                 </exclusion>
@@ -33,6 +37,10 @@
             <artifactId>openshift-annotations</artifactId>
             <classifier>noapt</classifier>
             <exclusions>
+                <exclusion>
+                    <groupId>io.sundr</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
                 <exclusion>
                     <groupId>com.sun</groupId>
                     <artifactId>tools</artifactId>

